app/src/main/java/com/halcyonmobile/android/common/extensions/application/ui/shared/BindingViewHolder.kt

```diff
@@ -0,0 +1,13 @@
+package com.halcyonmobile.android.common.extensions.application.ui.shared
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.recyclerview.widget.RecyclerView
+
+class BindingViewHolder<Binding : ViewDataBinding> constructor(val binding: Binding) : RecyclerView.ViewHolder(binding.root) {
```

Comment:
You may have wanted to use this, but ATM it's unused, so I think we should remove it for the sake of keeping the samples as simple as possible.